src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ClientRepository.cs

```diff
@@ -117,21 +184,46 @@ public async Task DeleteClientAsync(Guid clientId)
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"DELETE FROM Clients 
-                                 WHERE Id = @clientId";
+                var sqlQuery = $@"
+                    DELETE FROM Clients 
+                    WHERE 
+                        {nameof(Client.Id)} = @{nameof(clientId)}";
 
                 await connection.ExecuteAsync(sqlQuery, new { clientId });
             }
         }
-        
+
         public async Task<IReadOnlyCollection<Client>> GetClientsWithMembersAsync(List<Guid> memberIds)
         {
             using (var connection = _sqlConnectionFactory.CreateConnection())
             {
-                var sqlQuery = @"SELECT *
-                                 FROM Clients c 
-                                 INNER JOIN Members m ON c.MemberId = m.Id
-                                 WHERE c.MemberId IN @memberIds";
+                var sqlQuery = $@"
+                    SELECT
+                        c.{nameof(Client.Id)},
+                        c.{nameof(Client.ClientConnectionId)},
+                        c.{nameof(Client.LastActivity)},
+                        c.{nameof(Client.LastClientActivity)},
+                        c.{nameof(Client.Name)},
+                        c.{nameof(Client.UserAgent)},
+                        c.{nameof(Client.MemberId)},
+                        m.{nameof(Member.Id)},
+                        m.{nameof(Member.Email)},
+                        m.{nameof(Member.IsAfk)},
+                        m.{nameof(Member.IsBanned)},
+                        m.{nameof(Member.LastActivity)},
+                        m.{nameof(Member.LastNudged)},
+                        m.{nameof(Member.Name)},
+                        m.{nameof(Member.PhotoName)},
+                        m.{nameof(Member.Role)},
+                        m.{nameof(Member.SaasUserId)},
+                        m.{nameof(Member.Status)},
+                        m.{nameof(Member.IsActive)}
+                    FROM 
```

Comment:
When we'll merge pull request with new chat version, we need to be sure that we'll add new property:
m.{nameof(Member.IsDeleted)}

Reply:
We won't forget :)